test/__tests__/fsresolver.spec.ts

```diff
@@ -0,0 +1,81 @@
+jest.mock('fs');
+import * as path from "path";
+import { FsParser, FsResolver, SubParser, SubResolver } from "../../src";
+import { defaultContext } from "../utils";
+import { vol } from 'memfs';
+
+describe("FsResolver", function () {
+  let instance: SubResolver;
+  let contentsParser: SubParser<string>;
+
+  beforeAll(function () {
+    contentsParser = FsParser();
+    process.chdir(__dirname);
+  });
+
+  afterEach(function () {
+    vol.reset();
+  });
+
+  describe("without root prefix", function () {
+    beforeEach(function () {
```

Comment:
The `beforeEach` is the same for both root prefix and without root prefix. Move both upwards to the main `describe`

Reply:
Also might be worth moving into `beforeAll`